### PR TITLE
Add maxWidth and maxHeight rules

### PR DESCRIPTION
**Pull request by ksk5280**

### CODE REVIEW

File: src/rules/heights.ts [hunk 1]
```diff
@@ -1,30 +1,45 @@
-import { RuleFn } from "./RuleConfig";
-import { makeIncRules, makeRules } from "../utils";
+import { RuleFn } from './RuleConfig';
```
Comment (stephenh) [inline src/rules/heights.ts:1]: This should still be `"`?

File: src/rules/heights.ts [hunk 2]
```diff
@@ -1,30 +1,45 @@
...
   ...makeRules(
-    "minHeight",
+    'minHeight',
     {
       mh0: 0,
-      mh100: "100%",
-      mvh100: "100vh",
+      mh25: '25%',
+      mh50: '50%',
+      mh575: '75%',
```
Comment (stephenh) [inline src/rules/heights.ts:27]: An extra 5 I think.

Comment (ksk5280) [reply src/rules/heights.ts:27]: oops 😅 thanks!

File: src/rules/heights.ts (hunk 1 above)
Comment (ksk5280) [reply src/rules/heights.ts:1]: ah vscode and prettier auto saved those as single quotes. I'll revert back to double

**Pull request merged**